packages/wonder-blocks-button/src/components/__docs__/button.argtypes.js

```diff
@@ -66,15 +66,15 @@ export default {
     },
     size: {
         description: "The size of the button.",
-        options: ["small", "medium", "xlarge"],
+        options: ["small", "medium", "large"],
         control: {type: "select"},
         table: {
             category: "Layout",
             defaultValue: {
-                detail: `"medium" = height: 40; "small" = height: 32; "xlarge" = height: 60;`,
+                detail: `"medium" = height: 40; "small" = height: 32; "large" = height: 56;`,
```

Comment:
We're sure we'll never need a button with height `48`?

Reply:
Good question... that's what we ended up upon (@megaroeny in case you have any thoughts).